Ginger/GingerTest/APIModelLib/POMsTest.cs

```diff
@@ -144,7 +144,7 @@ public void ValidateElementsProperties()
             Assert.IsTrue(IsPropertyExist(ComboBoxEI.Properties,"X", "631"), "POM.property 6 check");
             Assert.IsTrue(IsPropertyExist(ComboBoxEI.Properties,"Y", "226"), "POM.property 7 check");
             Assert.IsTrue(IsPropertyExist(ComboBoxEI.Properties,"Value", "set to "), "POM.property 8 check");
-            Assert.IsTrue(IsPropertyExist(ComboBoxEI.Properties,"Optional Values", ",Ahhhh...,Got It!,Too far,OMG,"), "POM.property 9 check");
+            Assert.IsTrue(IsPropertyExist(ComboBoxEI.Properties,"Optional Values", "Ahhhh...*,Got It!,Too far,OMG,"), "POM.property 9 check");
```

Comment:
you need to add Assert methods to validate new list you added have all optionalValue by count and actual details, validate defualt value, etc.

Reply:
Added code